docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -7,6 +7,76 @@ uses:
   secrets: secrets.raml
 
 types:
+  CSIVolumeInfo:
+    type: object
+    properties:
+      name:
```

Comment:
We also need a string field `pluginName` which should be mapped to `Volume.csi_volume.plugin_name`, it can be `pxd.portworx.com` (Portworx CSI plugin) or `nfs.csi.k8s.io` (NFS CSI plugin) or the name of any 3rd party CSI plugin that user wants to use.